### DISCUSSION TITLE
Is it possible that a WebSocketServer sends a Message that triggers an Event

### DISCUSSION THREAD

**Discussion by ju-emb** [artisan-roaster-scope/artisan#1688] [General]
I have an Event Button, that sends a WebSocket command to switch on the sparcSource for the Gas.
The sparcSource stays on for 15 sec then switches off. 
That's not artisan managed, the underlying board does that.
However, the Board reflects the state of the sparcSource, but I'm wondering how do I get artisan to read that state (lets say in an interval) and switch the Button in the GUI off if the sparcSource is Off.

Thanks for helping out
ju

**Comment by MAKOMO** [comment]
I fear this is not possible with Artisan. You need to start developing your own software to support such machine-specific functionality.
